CMSgov/bluebutton-sample-client-nodejs-react
# jimfuqian/bump-micromatch-to-4.0.8-address-vulns: CVE-2024-4067 n 4068

**JIRA Ticket:**
[BB2-XXXX](https://jira.cms.gov/browse/BB2-XXXX)


### What Does This PR Do?

Bump micromatch to 4.0.8 to address (M) vulns per dependa bot


### What Should Reviewers Watch For?


If you're reviewing this PR, please check for these things in particular:



### Validation

1. Check out PR
2. Start in local and configure the sample app e.g. pointing to your SBX registered app
3. Do a authorization flow and see the claims for the synthetic user e.g., BBUser00001
4. After the PR merged the dependa bot will no longer pop up auto generated PR on these CVEs
